modules/mod_translation/lib/css/mod_translation.css

```diff
@@ -0,0 +1,145 @@
+.dropdown-menu.mod_translation_menu-has-icons > li > a {
+  padding-left: 40px;
```

Comment:
I cannot find an option to set the number of spaces or read from editorconfig. The recommended way for compression is to install the less plugin [clean-css](https://github.com/less/less-plugin-clean-css) and then call lessc with `--clean-css`. That puts all code on 1 lines.